Define consistent pylint output template for python2 and python3

**Impact of the new feature**
WMCore in general

**Is your feature request related to a problem? Please describe.**
It's been some time I noticed a different output format between py2 pylint and py3 pylint output. An example of their output is provided at the very bottom of this issue.

**Describe the solution you'd like**
Define a standard output template in the `.pylintrc` such that both python2 and python3 pylint outputs will have the same format.

**Describe alternatives you've considered**
Not change anything, or use default output for python3...

**Additional context**
Output of python2 pylint:
```
(PY2) AlanNewMacPro:WMCore amaltaro$ pylt src/python/Utils/IteratorTools.py
Using config file /Users/amaltaro/Pycharm/WMCore/standards/.pylintrc
************* Module src.python.Utils.IteratorTools
C:  1, 0: Missing module docstring (missing-docstring)
R: 47, 4: Unnecessary "else" after "return" (no-else-return)
C:  5, 0: standard import "from builtins import str, map" should be placed before "from future.utils import viewitems" (wrong-import-order)
C:  6, 0: standard import "import collections" should be placed before "from future.utils import viewitems" (wrong-import-order)
C:  7, 0: standard import "from itertools import islice, chain" should be placed before "from future.utils import viewitems" (wrong-import-order)
```

while this is the output for python3 pylint:
```
(PY3) AlanNewMacPro:WMCore amaltaro$ pylt src/python/Utils/IteratorTools.py
************* Module src.python.Utils.IteratorTools
src/python/Utils/IteratorTools.py:1:0: C0114: Missing module docstring (missing-module-docstring)
src/python/Utils/IteratorTools.py:47:4: R1705: Unnecessary "elif" after "return" (no-else-return)
src/python/Utils/IteratorTools.py:5:0: C0411: standard import "from builtins import str, map" should be placed before "from future.utils import viewitems" (wrong-import-order)
src/python/Utils/IteratorTools.py:6:0: C0411: standard import "import collections" should be placed before "from future.utils import viewitems" (wrong-import-order)
src/python/Utils/IteratorTools.py:7:0: C0411: standard import "from itertools import islice, chain" should be placed before "from future.utils import viewitems" (wrong-import-order)
```

--- a/standards/.pylintrc
+++ b/standards/.pylintrc
@@ -34,6 +34,13 @@ comment=no
 # Include message's id in output
 include-id=no
 
+[REPORTS]
+# Template used to display messages. This is a python new-style format string
+# used to format the message information. See doc for all details
+#msg-template='[{msg_id} {symbol}] {msg} File: {path}, line {line}, in {obj}'
+msg-template='{msg_id} L{line}: {msg} ({symbol})'
+
+
 [MESSAGES CONTROL]
 # disable F0401 - could not import module
 # disable E1103 - Allows attachment of dbi, logger to threading.currentThread()

